Lab1/Lab1.m

```diff
@@ -0,0 +1,150 @@
+%% A Little Statistics 
+
+%% Section 1B
+% Integrates a standard normal distribution using normcdf(x). These
+% values are approximately equal to the expected values listed on
+% the z-table on Wikipedia. 
+disp(normcdf(0))
+disp(normcdf(0.1))
+disp(normcdf(0.3))
+disp(normcdf(0.5))
+disp(normcdf(1))
+disp(normcdf(-1))
+
+%% Section 1C
+% Finds the associated sigma of the inverse of the standard normal 
+% distribution using norminv(x). These values are approximately 
+% equal to the expected value. 
+disp(norminv(0.84134))
+disp(norminv(0.97725))
+disp(norminv(0.99865))
+disp(norminv(0.99997))
+disp(norminv(0.00003))
+
+%% Section 1D
+% The negative sign from norminv(x) indicates the bounds of the integral
+% when calculating the probability of the distribution. This means the
+% bounds are from negative infinity to z rather than z to positive 
+% infinity. 
```

Comment:
I believe you will get a positive sigma for an integral form -infinity to any value past the mean or for probability above 0.5. I believe your sigma is how far form the mean you are. so you are outputting a sigma and it should not be in your bounds.

Reply:
I will describe how Matlab calculates the z value from probability. From the documentation, they use the given p value to find the z value by calculating an integral with bounds from -Inf to z. This is then put into inverse of z(p) to solve for z. 
![image](https://user-images.githubusercontent.com/72474045/95667267-839a9a00-0b18-11eb-9fc3-580ca437444e.png)